code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -656,6 +657,10 @@ Code:
 			loc:mode = 433
 			mode = 433
 
+		if("Supply Orders")
+			loc:mode =47

```

Comment:
Also holy shit there's some magic going on here; any way you could make them defines?